test/pylib/cql_repl/conftest.py

```diff
@@ -78,6 +78,14 @@ def cql(request):
                       auth_provider=PlainTextAuthProvider(username='cassandra',
                                                           password='cassandra'),
                       ssl_context=ssl_context,
+                      # The default timeout for new connections is 5 seconds, and for
+                      # requests made by the control connection is 2 seconds. These should
+                      # have been more than enough, but in some extreme cases with a very
+                      # slow debug build running on a very busy machine, they may not be.
+                      # so let's increase them to 60 seconds. See issue #11289.
+                      connect_timeout = 60,
+                      control_connection_timeout = 60,
+                      max_schema_agreement_wait=60,
```

Comment:
Good, this was the main missing place.
Again, please think if we need max_schema_agreement for what is basically a single-node test framework, and if we do, please explain it in the commit message.

Reply:
Fixed in https://github.com/scylladb/scylladb/commit/c1ebd7e95589bb313679f526c10045f6a1dcbe81